t/ui/15-admin-workers.t

```diff
@@ -124,6 +124,10 @@ subtest 'worker overview' => sub {
     # check worker 2
     is($driver->find_element('tr#worker_2 .worker')->get_text(), 'remotehost:1', 'remotehost:1 shown');
     $driver->find_element('tr#worker_2 .help_popover')->click();
+    for (0 .. 3) {
+       last if $driver->find_element('.popover')->get_text() =~ qr/Worker status\nJob: 99961/;
+       sleep 1;
```

Comment:
This is likely due to the popover animation. Better use `wait_for_element` here.

Reply:
The same as above - the can be done later, while failing tests cause problems now